sample_yaml/dpdk/pao.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: performance.openshift.io/v1
```

Comment:
Rename file to dpdk-performance-profile.yaml as pao.yaml is not very descriptive